exchange/bitswap/decision/peer_request_queue.go

```diff
@@ -125,6 +161,38 @@ func (tl *prq) Remove(k key.Key, p peer.ID) {
 	tl.lock.Unlock()
 }
 
+//  Should be called when ledger information for peer with pid is updated.
+//  Updates peers position in the partner queue.
+func (tl *prq) UpdatePeer(pid peer.ID) {
+	tl.lock.Lock()
+	defer tl.lock.Unlock()
+	if p, ok := tl.partners[pid]; ok {
+		tl.pQueue.Update(p.Index())
+	}
+}
+
+func (tl *prq) block(partner *activePartner) {
+	time.AfterFunc(PEER_BLOCK_TIME, func() {
+		tl.pQueue.Push(partner)
+	})
+}
+
+var Mean = func(partner *activePartner) bool {
```

Comment:
I was just mimicking the `var FIFO =` and `var V1 =` since I think it emphasizes that it's used as a variable.